fdbclient/FDBTypes.h

```diff
@@ -54,6 +55,33 @@ inline bool isPseudoLocality(int8_t locality) {
 	return locality == tagLocalityLogRouterMapped || locality == tagLocalityBackup;
 }
 
+#pragma pack(push, 1)
+namespace team {
+struct TeamID {
+	constexpr static FileIdentifier file_identifier = 345787;
+
+	uint32_t id;
```

Comment:
This needs more thought. We might need to use `UID` to capture more information, such as epochs, data center ID (locality).